android/compile.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
![like this?](http://www.funniestmemes.com/wp-content/uploads/2014/03/Funniest_Memes_ok-so-i-have-a-sick-sense-of-humor_8667-720x340.jpeg)